<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
text-decoration: underline;
font: 8pt &quot;MS Shell Dlg 2&quot;;
font: 75 16pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Consultas</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>531</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Consulta A</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetA">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Consulta B</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetB">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Consulta C</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetC">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Consulta D</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetD">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Consulta E</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetE">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Consulta F</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetF">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Consulta G</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetG">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>531</width>
       <height>481</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="BtnVolver">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
